core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -175,22 +178,84 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
+
+        contentsTable.clear()
+        
+        if (UncivGame.Current.settings.useDemographics) contentsTable.add(buildDemographicsTable(majorCivs))
+        else contentsTable.add(buildRankingsTable(majorCivs))
+    }
+
+    enum class RankLabels { Demographic, Rank, Value, Best, Average, Worst}
+    private fun buildDemographicsTable(majorCivs: List<CivilizationInfo>): Table {
+        val demographicsTable = Table().apply { defaults().pad(5f) }
+        for (rankLabel in RankLabels.values())   {
+            if (rankLabel == RankLabels.Demographic) {
+                val demoLabel = Table().apply { defaults().pad(5f) }
+                demoLabel.add(rankLabel.name.toLabel()).row()
+                demoLabel.addSeparator().fillX()
+                demographicsTable.add(demoLabel)
+                for (category in RankingType.values()) {
+                    val headers = Table().apply { defaults().pad(5f) }
+                    headers.add(category.name.replace('_', ' ').toLabel()).row()
+                    headers.addSeparator().fillX()
+                    demographicsTable.add(headers)
+                    }
+            } else {
+                demographicsTable.row()
+                demographicsTable.add(rankLabel.name.toLabel())
+
+                for (category in RankingType.values()) {
+                    @Suppress("NON_EXHAUSTIVE_WHEN") // RankLabels.Demographic treated above
+                    when (rankLabel) {
+                        RankLabels.Rank -> { // finds the current player's rank in each category
+                            val rank = majorCivs.filter{ it.isAlive() }.sortedByDescending { it.getStatForRanking(category) }.indexOfFirst { it.civName == gameInfo.currentPlayerCiv.civName } + 1
+                            demographicsTable.add(rank.toLabel())
+                        }
+
+                        RankLabels.Value -> { // finds the current player's value in each category
+                            demographicsTable.add(getCivGroup(gameInfo.currentPlayerCiv, ": " + gameInfo.currentPlayerCiv.getStatForRanking(category).toString(), playerCivInfo)).fillX()
+                        }
+                      
+                        RankLabels.Best -> { // finds civ with the best value in each category
+                            val bestCiv = majorCivs.filter{ it.isAlive() }.maxByOrNull { it.getStatForRanking(category) }!!
+                            demographicsTable.add(getCivGroup(bestCiv, ": " + bestCiv.getStatForRanking(category).toString(), playerCivInfo)).fillX()
+                        }
+                      
+                        RankLabels.Average -> { // calculates the average value in each category
+                            var totalScore = 0
+                            for (civ in majorCivs.filter{ it.isAlive()} ) totalScore += civ.getStatForRanking(category)
+                            val averageScore = totalScore / (majorCivs.filter{ it.isAlive() }.lastIndex + 1)
```

Comment:
`(majorCivs.filter{ it.isAlive() }.lastIndex + 1)` == `majorCivs.count { it.isAlive() }`?

Reply:
... and totalScore probably == `majorCivs.filter { it.isAlive() }.sumOf { it.getStatForRanking(category) }`?

Reply:
Yes indeed, thank you for these simplifications. :)

Reply:
> Yes indeed

I trust you _check_ such stuff in some form.

Reply:
I did. Got a couple of save files that I've been using to test. Everything seems to check out as far as I can tell.